awscli/customizations/emr/describecluster.py

```diff
@@ -65,20 +63,12 @@ def _find_master_public_dns(self, cluster_id, parsed_globals):
             session=self._session, cluster_id=cluster_id,
             parsed_globals=parsed_globals)
 
-    def _call(self, operation_object, parameters, parsed_globals):
-        # We could get an error from get_endpoint() about not having
-        # a region configured.  Before this happens we want to check
-        # for credentials so we can give a good error message.
-        result = []
-        if not self._session.get_credentials():
-            raise NoCredentialsError()
-        endpoint = operation_object.service.get_endpoint(
+    def _call(self, session, operation_name, parameters, parsed_globals):
```

Comment:
I wanted to remove this function entirely and just use ``emrutils.call()``, but this internal method gets mocked out in tests and is used to provide mocked responses...
I can go back and fix the tests but I was trying to make the minimal amount of changes necessary especially for the tests.